src/components/HookForm/index.jsx

```diff
@@ -508,61 +531,85 @@ export default class HookForm extends Component {
               />
             </FormGroup>
           </ListItem>
-          {!isNewHook && (
-            <Fragment>
-              <ListItem>
-                <ListItemText
-                  primary="Last Fired"
-                  secondary={
-                    lastFireTypeName === HOOKS_LAST_FIRE_TYPE.NO_FIRE ? (
-                      'n/a'
-                    ) : (
-                      <DateDistance from={hook.status.lastFire.time} />
-                    )
-                  }
-                />
-              </ListItem>
-              {lastFireTypeName === HOOKS_LAST_FIRE_TYPE.SUCCESSFUL_FIRE ? (
-                <ListItem
-                  button
-                  component={Link}
-                  className={classes.listItemButton}
-                  to={`/tasks/${hook.status.lastFire.taskId}`}>
-                  <ListItemText
-                    primary="Last Fired Result"
-                    secondary={hook.status.lastFire.taskId}
-                  />
-                  <LinkIcon />
-                </ListItem>
-              ) : (
-                <ListItem>
-                  <ListItemText
-                    primary="Last Fired Result"
-                    secondary={
-                      lastFireTypeName === HOOKS_LAST_FIRE_TYPE.NO_FIRE ? (
-                        'n/a'
-                      ) : (
-                        <pre>
-                          {JSON.stringify(hook.status.lastFire.error, null, 2)}
-                        </pre>
-                      )
-                    }
-                  />
-                </ListItem>
-              )}
-              <ListItem>
+          <ListItem>
+            {hookLastFires ? (
+              <List className={classes.hookLastFires}>
                 <ListItemText
-                  primary="Next Scheduled Fire"
-                  secondary={
-                    hook.status.nextScheduledDate ? (
-                      <DateDistance from={hook.status.nextScheduledDate} />
-                    ) : (
-                      'n/a'
-                    )
+                  primary={
+                    <Typography variant="subtitle1">
+                      Last Hook Fire Attempts
+                    </Typography>
                   }
                 />
-              </ListItem>
-            </Fragment>
+                {hookLastFires.map(
+                  ({ taskId, taskCreateTime, firedBy, result, error }) => (
+                    <ListItem key={taskId}>
```

Comment:
Let's use the `DataTable` component in `src/components/DataTable` to display the result? The table will be  similar to something like http://localhost:5080/provisioners/aws-provisioner-v1.

Reply:
I don't think a DataTable would be a good fit for the data. If result is  `success` then expandable panel has nothing to show. If result is `error` then there is very large error text  to show on opening the panel. In  any column it could be trouble to fit the error text especially.